net.sourceforge.GTKWave.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 Tony Bybell <bybell@rocketmail.com> -->
+<component type="desktop">
+  <id>net.sourceforge.gtkwave.desktop</id>
```

Comment:
```suggestion
  <id>net.sourceforge.GTKWave</id>
```

`.desktop` should never be here. it's a deprecated thing.

Reply:
Oh sorry. I was looking at the logs and I saw some .desktop comparison in the regex and thought it was necessary.

Reply:
the regexp tolerate the deprecated format.